docs/design/modules/digital_twin_graph/README.md

```diff
@@ -0,0 +1,65 @@
+# Design Specification for Digital Twin Graph Service
+
+- [Introduction](#introduction)
+- [Architecture](#architecture)
+- [Operations](#operations)
+
+## <a name="introduction">Introduction</a>
+
+Ibeji today provides the foundations for constructing and interacting with a digital twin on an edge device. These are low-level abilities and do not necessarily
+provide a consumer with the best interaction experience. They can be used as building blocks to build facades that provide a consumer with an abstraction that
+delivers a much better interaction experience. In the future, Ibeji may support multiple facades and the user can select the one that they prefer to use.
+
+This design specifies a graph-based facade, which will be named the Digital Twin Graph Service. With this facade, the digital twin will be represented as a
+graph of digital twin entities whose arcs represent the relationships between those entities.  Instance ids will be used to refer to entities.
```

Comment:
I've never seen the word "arcs" used to refer to this. Perhaps "edge" is the more familiar term?

Reply:
Please see: https://en.wikipedia.org/wiki/Directed_graph

Reply:
I'm aware that the term is correct, but in computing I think "edge" is the more common for this construct in graphs regardless of directionality. This is not something that needs to be changed, but using "edge" could make it easier to understand. Even the article you linked states the following:

>connected by directed [edges](https://en.wikipedia.org/wiki/Edge_(graph_theory)), often called arcs

Reply:
I'm happy with either.  I'll change it to what you prefer.